kernel/common/range_mirror.rb

```diff
@@ -17,6 +17,8 @@ def step_float_iterations_size(first, last, step_size)
         else
           iterations = ((last - first) / step_size + err).floor + 1
         end
+
+        iterations
```

Comment:
Alternatively, this could be pushed inside the `if` block.  The issue only affects the `excl` case.